stdlib/socket/src/main/ballerina/src/socket/Module.md

```diff
@@ -1,154 +1,56 @@
-## Module overview
+This module provides an implementation for sending/receiving messages to/from another application process (local or remote) for both connection-oriented and connectionless protocols.
```

Comment:
```suggestion
This module provides an implementation for sending/receiving messages to/from another application process (local or remote) for both connection-oriented and connectionless protocols. The module facilitates a TCP `Client` and a TCP `Listener`.
```

Reply:
Connection-oriented and connectionless protocols indicate TCP and UDP. So, I think, Don't need to update this.